lib/RequestLogger.js

```diff
@@ -1,17 +1,19 @@
-'use strict'; // eslint-disable-line strict
+
+// eslint-disable-line strict
 
 const assert = require('assert');
 
 const LogLevel = require('./LogLevel.js');
 const Utils = require('./Utils.js');
-const serializeUids = Utils.serializeUids;
-const generateUid = Utils.generateUid;
-const objectCopy = Utils.objectCopy;
+
+const { serializeUids } = Utils;
+const { generateUid } = Utils;
+const { objectCopy } = Utils;
```

Comment:
why not this,
```suggestion
const { serializeUids, generateUid, objectCopy } = Utils;
```

Reply:
agree, @naren-rajendran  - all this changes are simply due to commits #c61c0a3 and #8d8c7c8 (autolinting) - so I didn't really go into detail on every line of code. I will add the suggested changes, thx

Reply:
Did not see this addressed